2-deprecate-big-bang/README.md

```diff
@@ -0,0 +1,280 @@
+
+To simplify the deployment of Big Bang within Zarf the Big Bang extension was created. This allowed users to create their Big Bang component in as little as three lines of code - version is the only required field. Below is the extensions key with a value for every sub-key.
+
+```yaml
+extensions:
+bigbang:
+    version: 2.34.0
+      skipFlux: false
+    repo: https://repo1.dso.mil/big-bang/bigbang.git
+    FluxPatchFiles:
+      - config/flux-patch.yaml
+    valuesFiles:
+      - config/values.yaml
+```
+
+During `zarf package create` the Big Bang extension is processed so that the created package includes everything needed for a Big Bang deployment. The zarf.yaml within the package tar contains a component with all the necessary images, repos, manifests and actions. To view the full component created by the Big Bang extension see [#2875](https://github.com/zarf-dev/zarf/issues/2875).
+
+### Problems with the Big Bang extension
+
+The extension does simplify deploying Big Bang in Zarf, but this comes with downsides.
+
+- The extension must be considered during package create, skeleton package create, and component compose increasing complexity in the codebase.
+- Zarf performs implicit actions that are not visible to the user, such as adding a custom values file with Kyverno policies.
+- Big Bang relies on images stored in [Iron Bank](https://p1.dso.mil/services/iron-bank), a container registry run by the DoD. Unfortunately, Iron Bank has frequent outages which causes flakes in Zarf’s test suite.
+
+A notable reason to move away from offering Big Bang support in Zarf is to focus on being an air-gap Kubernetes tool, rather than a DoD deployment tool. The Big Bang support exists because Zarf was created by the DoD contractor Defense Unicorns. Now Zarf has been donated to Open SSF and it should focus on what is best for the community at large.
+
+
+### Goals
+
+- Remove all references to Big Bang within Zarf.
+
+## Proposal
+
+The proposed solution is to create a new go project, [github.com/defenseunicorns-partnerships/generate-big-bang-zarf-package](https://github.com/defenseunicorns-partnerships/generate-big-bang-zarf-package), focused solely on generating a big bang package. The command `generate-big-bang-zarf-package` will accept have one argument, version, and accept the following command line flags: `values-file-manifests`, `skipFlux`, and `repo`. The `fluxPatchFiles` flag will not have an equivalent, however, users will be able to edit their Flux component to deploy it with patch files after the generate command has run. Below is the helper text for the command: 
```

Comment:
@schristoff this doesn't exist yet, but it is the intended link right ? https://github.com/defenseunicorns-partnerships/generate-big-bang-zarf-package

Reply:
Yup it is